include/aos/common/config.hpp

```diff
@@ -471,8 +471,18 @@
 #define AOS_CONFIG_CRYPTOUTILS_DEFAULT_PKCS11_LIB "/usr/lib/softhsm/libsofthsm2.so"
 #endif
 
+/**
+ * Maximum number of public keys to be allocated by cryptoprovider simultaneously.
+ */
 #ifndef AOS_CONFIG_CRYPTOPROVIDER_PUB_KEYS_COUNT
 #define AOS_CONFIG_CRYPTOPROVIDER_PUB_KEYS_COUNT 5
 #endif
 
+/**
+ * Size of a time in string representation.
+ */
+#ifndef AOS_TIME_STRING_LENGTH
```

Comment:
Should be in `common/tools/config.hpp` and named `AOS_CONFIG_TIME_STR_LEN`

Reply:
fixed